types/stylelint/index.d.ts

```diff
@@ -115,6 +117,10 @@ declare namespace stylelint {
 		configurationComment?: string;
 		overrides?: ConfigOverride[];
 		customSyntax?: CustomSyntax;
+		processors?: ConfigProcessors;
+		_processorFunctions?: {
+			[processorName: string]: Omit<ReturnType<Processor>, 'name'>;
+		}; // Internal use only
```

Comment:
[suggest]

- How about using `Map` instead of `Object` because `Map` has a more explicit property order than `Object`? ([ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#objects_vs._maps))
- How about using `@internal` as other internal types?

```suggestion
		/** @internal */
		_processorFunctions?: Map<string, ReturnType<Processor>['postprocess']>;
```

Reply:
Sounds good :) Yep agree keeping the order is better